README.md

```diff
@@ -49,7 +49,11 @@ To run the development environment:
   ```sh
   yarn install
   ```
-4. Run the development environment, which starts a server listening on [http://localhost:3020](http://localhost:3020):
+4. Run build
```

Comment:
Hi @dsaouda . Thanks for the contribution. There are some errors indeed when you run by the first time, but that doesn't break the development environment.  Perhaps it would make more sense to point out that the build script needs to be ran only by the first time. Can you update that?

```suggestion
4. If you are running the project by the first time, run the build script
```